virttest/utils_tmux.py

```diff
@@ -0,0 +1,23 @@
+"""
+The tmux command related utility functions
+"""
+
+import logging
+
+from virttest import utils_misc
+from virttest import utils_package
+
+
+def run_tmux_cmd(cmd, session=None):
+    """
+    Execute shell-commands in tmux on localhost or remote
+
+    :param cmd: The command line to run
+    :param session: The session object to the host
+    """
+    if not utils_package.package_install("tmux", session):
+        logging.error("Failed to install required package - tmux!")
```

Comment:
If tmux is necessary package , better raise exception directly here.

Reply:
Actually, I didn't not raise an exception here on purpose, the output of tmux command line will show the details.

Reply:
a little confused here, if tmux is not installed successfully and program doesn't terminate here, that means utils_misc.cmd_status_output will be executed without tmux command existence

Reply:
Yes, I'd like to check the result of utils_misc.cmd_status_output's return.